test/index.test.js

```diff
@@ -505,6 +505,27 @@ describe('Segment.io', function() {
             analytics.stub(segment, 'enqueue');
           });
 
+          it('identify should not ultimately call getCachedCrossDomainId if crossDomainAnalytics is not enabled', function() {
+            var wasCalled = false;
+            var oldCrossDomainCheck = segment.isCrossDomainAnalyticsEnabled;
+            segment.isCrossDomainAnalyticsEnabled = function() {
+              return false;
+            };
+            var oldGetCrossDomainId = segment.getCachedCrossDomainId;
+            segment.getCachedCrossDomainId = function() {
```

Comment:
Could/should we use `sinon` for ? I think the spies are less likely 

https://sinonjs.org/releases/v7.4.1/spy-call/

```
segment.options.crossDomainIdServers = [];

var getCachedCrossDomainIdSpy = sinon.spy(segment, "getCachedCrossDomainId");

segment.normalize({});

sinon.assert.notCalled(getCachedCrossDomainIdSpy);

segment.getCachedCrossDomainId.restore();
```

Reply:
done!